src/Dropdown/Dropdown.js

```diff
@@ -143,7 +136,7 @@ export default class Dropdown extends React.Component {
     const renderFn = render || children;
 
     return (
-      <DropdownWrapper tabIndex={0} onBlur={this.handleBlur}>
+      <div tabIndex={0} onBlur={this.handleBlur}>
```

Comment:
did you mean to rm `DropdownWrapper`? looks like you removed `DropdownTrigger` which is still in use

Reply:
Thanks @choochootrain, fixed.